code/game/atoms.dm

```diff
@@ -1392,6 +1394,24 @@
 
 		SEND_SIGNAL(src, COMSIG_ATOM_VV_MODIFY_TRANSFORM)
 
+	if(href_list[VV_HK_SPIN_ANIMATION] && check_rights(R_VAREDIT))
+		var/num_spins = input(usr, "How many spins?", "Spin Animation") as null|num
+		if(!num_spins)
+			return
+		var/spin_speed = input(usr, "How fast?", "Spin Animation") as null|num
+		if(!spin_speed)
+			return
+		var/direction = input(usr, "Which direction? (-1 = counter-clockwise, 1 = clockwise)", "Spin Animation") as null|num
```

Comment:
this is silly. make this a normal input where you click clockwise or counter clockwise

Reply:
Smoothbrained it and didn't do this for whatever reason. Thanks for the call out.